### ISSUE TITLE
Issues with the Support library v25

### ISSUE THREAD

**Issue by MisterY** [spacecowboy/NoNonsense-FilePicker#117]
There are a couple of blocker issues after upgrading the Support Library:

- merging the AndroidManifest.xml files fails and requires adding tools:replace="android:icon" to the application tag
- Error:(8462) Error retrieving parent for item: No resource found that matches the given name 'NNF_BaseTheme'.

After resolving the first issue, the second turns out to be a blocker. Any ideas? Thanks!

**Comment by MisterY** [comment]
Oh, it seems that the picker was downgraded to 1.2.0 in the process. Adjusting to v3.1.0 solves it.

**Comment by MisterY** [comment]
The underlying cause is that the dependency-update scripts reports the following:

```
The following dependencies exceed the version found at the milestone revision level:
 - com.nononsenseapps:filepicker [3.1.0 <- 1.2.0]
```

**Comment by spacecowboy** [comment]
@MisterY I'm not sure which dependency-update scripts you are referring to. Is there anything you or I can do here to make it easier for users?

**Comment by MisterY** [comment]
Hi!

Yes, it took me a while to find and remember where I got the script originally. Basically, it is the [versions plugin](https://github.com/ben-manes/gradle-versions-plugin) for Gradle.
It is added to the project via
```
plugins {
    // dependencyUpdates
    id 'com.github.ben-manes.versions' version '0.13.0'
}
```
effectively adding a Gradle task that checks for the updated versions of all dependent libraries. For some reason, it shows the filepicker only available as 1.2.0. This is quite strange since I think it told me about the version 3.1.0 a couple of weeks ago. 
Perhaps this would give you an idea of what may have gone wrong with the Maven distribution.

**Comment by spacecowboy** [comment]
I suspect the plugin is using maven central. 1.2.0 is latest version available on maven central:
http://search.maven.org/#search%7Cga%7C1%7Cnononsenseapps

all later releases are published to jcenter.

**Comment by MisterY** [comment]
Ah, that explains it. I will create an issue there and see if jcenter could be used as the source. Thanks!

**Comment by spacecowboy** [comment]
Bintray didn't seem to offer a way to edit or remove that file. 

Working on the proper fix now so should have the real 4.0 out today or tomorrow so this should be fixed then.
